app/controllers/application_controller.rb

```diff
@@ -2461,7 +2489,10 @@ def find_by_id_filtered(db, id)
     end
 
     Rbac.filtered(db, :conditions => ["#{db.table_name}.id = ?", id], :user => current_user).first ||
-      raise("User '#{current_userid}' is not authorized to access '#{ui_lookup(:model => db.to_s)}' record id '#{id}'")
+      raise(_("User '%{old_id}' is not authorized to access '%{model}' record id '%{new_id}'") %
+              {:old_id => current_userid,
+               :new_id => id,
```

Comment:
Just noticed in passing, but `old_id` and `new_id` seem rather unlikely names for these placeholders :confused: